README.md

```diff
@@ -42,13 +42,38 @@ Add the following attribute to the markup to enable a comment count:
 <div class="o-comments"
 -	id="o-comments-stream"
 	data-o-component="o-comments"
-	data-o-comments-article-id="{article-id}"
+	data-o-comments-article-id="{article-id}"   
```

Comment:
Looks like you have added some whitespace here

Reply:
Now removed